easybuild/easyconfigs/s/seqtk/seqtk-sgdp-intel-2015a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'seqtk'
+version = 'sgdp'
+
+homepage = 'https://github.com/lh3/seqtk/'
+description = """Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. 
+ It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/lh3/seqtk/archive/']
+
+buildopts = 'CC="$CC" CFLAGS="$CLFAGS"'
+
+files_to_copy = ["seqtk", "trimadap"]
+
+sanity_check_paths = {
+    'files': files_to_copy,
+    'dirs': [],
+}
+
+modextrapaths = {
+    'PATH': ['$root'],
```

Comment:
please use `%(installdir)s` here; `$root` is specific to Tcl modules, and we're close to adding support for Lua module files